tests/sqllogictests/suites/base/05_ddl/05_0019_ddl_create_view

```diff
@@ -111,3 +111,39 @@ AS
     SELECT *
     FROM
         's3://testbucket/admin/data/tuple.parquet' (FILES => ('tuple.parquet','test.parquet'),FILE_FORMAT => 'parquet',PATTERN => '.*.parquet', aws_key_id => 'minioadmin', aws_secret_key => 'minioadmin', endpoint_url => 'http://127.0.0.1:9900/')
+
+statement ok
+drop view if exists loop_view1;
+
+statement ok
+drop view if exists loop_view2;
+
+statement ok
+drop view if exists loop_view3;
+
+statement ok
+create view loop_view1 as select * from loop_view3;
```

Comment:
Can we check the loop when create view ?

```sql
postgres=# create view v_t as select * from v_t;
ERROR:  relation "v_t" does not exist
LINE 1: create view v_t as select * from v_t;

```

Reply:
I'm afraid not in current implementation. Because when creating a view, we just store the SQL string.